### ISSUE TITLE
Recommended pretraining epochs for micro-scale model to match downstream results.

### ISSUE THREAD

**Issue by yao-w3** [lbcb-sci/RiNALMo#26]
Hi, thank you for sharing this great work!

I have a question about pretraining the micro-scale model.
How many epochs of pretraining are typically required to achieve the downstream performance shown in the paper?
Is there a recommended number of epochs, or any indicator to determine when the model is sufficiently pretrained for those benchmark tasks?

**Comment by RJPenic** [comment]
In general, you usually want to pre-train the model until you notice stagnation of the pre-training loss (or until you spend all your computational resource "credits" allocated for pre-training).

Do keep in mind that `micro` model achieved relatively limited results in downstream tasks (compared to larger configurations), so do not expect stellar results if you decide to play around with it. 😅
